WPObjectMapper/NSObject+ObjectMap.m

```diff
@@ -563,7 +563,8 @@ +(NSArray *)arrayMapFromArray:(NSArray *)nestedArray forPropertyName:(NSString *
             }
             
             // Finally add that object
-            [objectsArray addObject:nestedObj];
+            if (nestedObj)
```

Comment:
Is that because you get a `nil` here?

https://github.com/WPMedia/ios-object-mapper/pull/9/files#diff-3149f13d3147ec363b5846371d04d7adR500

Why would that be?